### PR TITLE
Fix#1

### PR DESCRIPTION
**Pull request by AyonCh**
Hmm do check if this works before pushing it

### CODE REVIEW

File: events/message.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-  execute: async (message) => {
-    prefix = "t!" || "t~";
+    name: "message",
+    once: false,
+    execute: async (message) => {
+        prefix = "t!" || "t~";
 
-    const args = message.content.slice(prefix.length).trim().split(/ +/g);
+        const args = msg.startsWith(prefix)
+            ? msg.slice(prefix.prefix).trim().split(/ +/g)
```
Comment (kavinvalli) [inline events/message.js]: `prefix` is a string. I don't think `prefix.prefix` is a thing.

**Review by kavinvalli — CHANGES_REQUESTED** [review]
Also, fix the formatting please.

File: events/message.js (hunk 1 above)
Comment (AyonCh) [reply events/message.js]: ya typing mistake is should have been length

**Review by kavinvalli — APPROVED** [review]
lgtm merging

**Pull request merged**